demisto_sdk/commands/lint/linter.py

```diff
@@ -296,14 +299,13 @@ def _run_lint_in_host(self, no_flake8: bool, no_bandit: bool, no_mypy: bool, no_
         Args:
             no_flake8(bool): Whether to skip flake8.
             no_bandit(bool): Whether to skip bandit.
-            no_mypy(bool): Whether to skip mypy.
```

Comment:
why did you remove it? it still an argument

Reply:
@yaakovi - no mypy on host here.